proto/regen/ecocredit/basket/v1/events.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package regen.ecocredit.basket.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "regen/ecocredit/basket/v1/types.proto";
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit/basket/v1";
+
+// EventCreate is an event emitted when a basket is created.
+message EventCreate {
+
+  // basket_denom is the basket bank denom.
+  string basket_denom = 1;
+
+  // curator is the address of the basket curator who is able to change certain
+  // basket settings.
+  string curator = 2;
+
+  // display_name is used to create the bank denom metadata display name for this
+  // basket token.
+  string display_name = 3;
+
+  // exponent is the exponent that will be used for converting credits to basket
+  // tokens and for bank denom metadata.
+  uint32 exponent = 4;
+
+  // disable_auto_retire allows auto-retirement to be disabled.
+  bool disable_auto_retire = 5;
+
+  // credit_type_name filters against credits from this credit type name.
+  string credit_type_name = 6;
+
+   // allowed_classes are the credit classes allowed to be put in the basket.
+  repeated string allowed_classes = 7;
+
+  // min_start_date is the earliest start date for batches of credits allowed
+  // into the basket.
+  google.protobuf.Timestamp min_start_date = 8;
+
+  // fee is the fee that the curator paid to create the basket.
+  repeated cosmos.base.v1beta1.Coin fee = 9 [
+    (gogoproto.nullable) = false,
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+  ];
```

Comment:
I would keep only:
+ basket_denom
+ creator (msg sender)
+ curator
+ (maybe) min_start_date

Reply:
I'm not sure I understand. Why curator twice? I think the other information might be valuable but we can also remove some if not necessary. What do others think?

Reply:
curator is the same as creator. I think we should just have basket_denom + curator and keep it simple. the other data can be queried

Reply:
Ok, simplified to include:

- owner
- basket_denom
- credits
- amount (tokens converted to credits)